docs/articles/documentation/using-testcafe/common-concepts/using-browsers.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: docs
+title: Using Browsers
+permalink: /documentation/using-testcafe/common-concepts/using-browsers.html
+---
+# Using Browsers
+
+TestCafe comes with a [built-in library](#built-in-library) and several [plugins](#browser-provider-plugins) that allow you to easily set up testing in different web browsers.
+
+## Built-in Library
+
+The [testcafe-browser-natives](https://github.com/DevExpress/testcafe-browser-natives) built-in library is used to work with popular local browsers during testing: to open and close them, to take screenshots of browser windows, etc.
+To run tests in these browsers, you just need to specify them, and TestCafe will perform all required actions on browser windows by itself.
+
+You can select browsers in one of two ways:
+
+* through the command line (see [Local Browsers](../command-line-interface.md#local-browsers))
+* programmatically (by using the [runner.browsers](../programming-interface/runner.md#browsers) method)
+
+### Browser Names
+
+To identify browsers, browser short names are used. They allow you to avoid dealing with paths to browser executables or other awkward ways to refer to a browser.
+
+The following table lists all available browser names.
+
+Browser           | Alias
+----------------- | ----------
+Chromium          | `chromium`
+Google Chrome     | `chrome`
+Internet Explorer | `ie`
+Microsoft Edge    | `edge`
+Mozilla Firefox   | `firefox`
+Opera             | `opera`
+Safari            | `safari`
+
+To obtain the list of browsers installed on the current machine, run TestCafe
+from the command line with the [-b option](../command-line-interface.md#-b---list-browsers).
+
+## Browser Provider Plugins
+
+TestCafe provides plugins for testing in [Sauce Labs](https://saucelabs.com/) and [PhantomJS](http://phantomjs.org/) browsers:
```

Comment:
We don't provide them out of the box, so we should show them here as an example, also we need to mention how to find other plugins (using link to npm search with appropriate query params, like we did for reporters)